script/update_marriage_abroad_opposite_sex_outcomes.rb

```diff
@@ -98,7 +98,7 @@ def insert_above_heading(opposite_sex_paths, variation, text)
       puts "=== Changes to #{country} opposite sex outcomes, using variation #{variation} ==="
 
       Find.find(country_dir) do |path| # find outcomes for country
-        opposite_sex_paths << path if path =~ /_opposite_sex.govspeak.erb/
+        opposite_sex_paths << path if path =~ /_opposite_sex.*.erb/
```

Comment:
This ought be fixed up to the first commit. Partly for tidiness, but partly because [if someone comes back to grab the script from version control](https://github.com/alphagov/smart-answers/pull/4534#discussion_r428507203), we want to give them a fully working script 👍